src/main/java/org/isf/permissions/mapper/PermissionMapper.java

```diff
@@ -39,27 +37,11 @@ public PermissionMapper() {
 
 	@Override
 	public List<PermissionDTO> map2DTOList(List<Permission> list) {
-		return list.stream().map(permission -> {
-			PermissionDTO dto = modelMapper.map(permission, PermissionDTO.class);
-			List<String> userGroupCodes = extractUserGroupCodes(permission);
-			dto.setUserGroupIds(userGroupCodes);
-			return dto;
-		}).collect(Collectors.toList());
+		return list.stream().map(permission -> modelMapper.map(permission, PermissionDTO.class)).collect(Collectors.toList());
 	}
 
 	@Override
 	public PermissionDTO map2DTO(Permission fromObj) {
-		PermissionDTO dto = modelMapper.map(fromObj, PermissionDTO.class);
-		List<String> userGroupCodes = extractUserGroupCodes(fromObj);
-		dto.setUserGroupIds(userGroupCodes);
-		return dto;
-	}
-
-	private List<String> extractUserGroupCodes(Permission fromObj) {
-		if (fromObj.getGroupPermission() == null) {
-			return new ArrayList<>();
-		}
-		return fromObj.getGroupPermission().stream().map(gp -> gp.getUserGroup().getCode()).collect(Collectors.toList());
+		return modelMapper.map(fromObj, PermissionDTO.class);
```

Comment:
same comment as above.

It would be a great service to everyone if there were some tests for the `permissions` package in this repo;  Currently there are none.

Reply:
@SilverD3 why is not possible to have permissions JUnit tests in this PR itself?

Reply:
It's possible. It's just that there's another issue (OP-1309) I've started in which I'm going to modify the permissions endpoints, and I think it would be more practical to write the tests in that other task.

Reply:
I see... ok then!